lib/rack/query_parser.rb

```diff
@@ -129,6 +129,8 @@ def normalize_params(params, name, v, _depth=nil)
 
       return if k.empty?
 
+      v ||= String.new
```

Comment:
At one point, I think we made this configurable using a method - is that still worth considering? I liked that approach.

Reply:
I don't think so.  There isn't a strong need for it now that form_pairs can be used for custom parameter parsing.